### PR TITLE
Adding Canvas Project

### PR DESCRIPTION
**Pull request by anayatkhan1**
# Zero To Mastery - Canvas 2022 🎉🎉

## PR contains:
- Hey, I'm a Active contributor. I have done Canvas Project (MousyBubble) for Hacktoberfest 2022, it is a Canvas Animation i hope you like it my contribution 🤜

## Breaking changes
-  None

## Screenshots
icons

### CODE REVIEW

**Comment by anayatkhan1** [comment]
is it important to keep the js file name is art.js, because i actually make this file name is script.js. is everything is alright

File: src/art/AnayatCanvas/script.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    context.strokeStyle = "blue";
+    context.stroke();
+    
+}
+// Calling Function
+
+init();
+Animate();
+
+// Below Code is for practicing for making animation, canvas and So on.
```
Comment (sophiabrandt) [inline src/art/AnayatCanvas/script.js:149]: Please remove unused code

File: src/art/AnayatCanvas/styles.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+/* canvas{
```
Comment (sophiabrandt) [inline src/art/AnayatCanvas/styles.css:2]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: src/art/AnayatCanvas/script.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    var dy = (Math.random() - 0.5) * 3;
+    var radius = Math.random() * 3 + 1;
+
+    circleArray.push(new Circle(x,y, dx, dy, radius));
+    
+}
+
+}
+
+// console.log(circleArray);
```
[the remark this answers is not in the document]
Comment (anayatkhan1) [reply src/art/AnayatCanvas/script.js]: i remove the unused code

**Pull request merged**